python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -340,6 +343,7 @@ def encode_env_var(value: Any) -> str:
     Returns:
         str: The encoded value.
     """
+    _json_serialize_param(value, "encode_env_var", "value")
```

Comment:
why add the return type if don't use it?

Reply:
was a mistake, meant to rm next line, fixed